aws_ses.go

```diff
@@ -11,8 +11,8 @@ import (
 
 // awsSesInterface is an interface for ses/mocking
 type awsSesInterface interface {
-	SendEmail(from, to, subject, body string) (string, error)
-	SendEmailHTML(from, to, subject, bodyText, bodyHTML string) (string, error)
+	//SendEmail(from, to, subject, body string) (string, error)
+	//SendEmailHTML(from, to, subject, bodyText, bodyHTML string) (string, error)
 	SendRawEmail(raw []byte) (string, error)
```

Comment:
The SendEmail and SendEmailHTML APIs are commented out simply because they are not used here. Those APIs are still on go-ses, however they have different signatures:
```
func SendEmailHTML(from string, to, cc, bcc []string, subject, bodyText, bodyHTML string) (string, error) 
func SendEmail(from string, to, cc, bcc []string, subject, body string) (string, error) 
```